docs/PersonalApplications.rst

```diff
@@ -1285,9 +1285,33 @@ You will receive the **My News shared** notification when a user shares one of y
 
 	|image99|
 
-**My News liked**
+**My posted News liked**
```

Comment:
"My liked posted News" ?

Reply:
or "My liked news"

Reply:
Shall we put the exact text we have in the page notifications settings? @plamarque @thomasdelhomenie

Reply:
well, make the change in product :p